lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/security/SoapTokenRetriever.java

```diff
@@ -34,7 +43,49 @@ public String getSoapToken() {
             }
             retry++;
         }
-        return token;
+
```

Comment:
Line 42:  `LOGGER.error("Error while getting soap token", e.getMessage());` should be fine

Reply:
doing `e` gives us the stacktrace which is helpful for debugging